README.md

```diff
@@ -67,6 +67,27 @@ You can now use the high-fidelity tools to tweak your CSS and inspect network ca
 
 ![Microsoft Edge Tools - Demo](basic_usage.gif)
 
+
+#### Setting up your project to show live changes in the extension
+
+Out of the box the browser does not track live changes to the code you write. If you want the extension to automatically refresh the browser when you changed a file, you need to set up a live reload environment. For this you need Node.js and npm on your machine. 
+
+Say you have a folder with your production files on your hard drive called `my-project`. 
+
+Preparation step: Install Node.js and the reload package
+
+* Download and install [Node.js](https://www.nodejs.org) (you only need to do this once).
+* Install the [reload NPM package](https://www.npmjs.com/package/reload?activeTab=readme)
+    - Open command prompt and run `npm install reload -g` to install the package globally
+
+Attach the extension to your live reloading project
+
+* Navigate to your `my-project` folder in your command prompt and run `reload`
+* Open VS Code and open the directory
+* Go to the extension and launch an instance
+* Navigate to localhost:8080/{file name you want to open}	
```

Comment:
NIT: change to code style text (e.g. `localhost:8080/{file name you want to open}`)

Reply:
Fixed, I forgot to commit that.